vms/avm/service_test.go

```diff
@@ -538,10 +538,13 @@ func TestServiceGetTxJSON_BaseTx(t *testing.T) {
 	replyTxBytes, err := json.MarshalIndent(reply.Tx, "", "\t")
 	require.NoError(err)
 
-	expectedReplyTxString := `{
+	sigStr, err := formatting.Encode(formatting.HexNC, newTx.Creds[0].Credential.(*secp256k1fx.Credential).Sigs[0][:])
+	require.NoError(err)
+
+	expectedReplyTxString := fmt.Sprintf(`{
 	"unsignedTx": {
 		"networkID": 10,
-		"blockchainID": "PLACEHOLDER_BLOCKCHAIN_ID",
+		"blockchainID": "%s",
```

Comment:
`"%s"` can (and usually should) be replaced with `%q`, which will also escape inputs as necessary. This applies throughout the PR.

Reply:
Good call, replaced.